src/main/java/mate/controller/driver/GetAllDriversController.java

```diff
@@ -0,0 +1,35 @@
+package mate.controller.driver;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers")
```

Comment:
It is a good practice to provide a more descriptive URL pattern for the servlet. Consider changing the URL pattern to '/drivers/all' to better represent the functionality of this controller.

Reply:
But in "Common mistakes (jv-web-practice)"
Good URLs naming in your project - /drivers - (get all drivers)
Bad URLs naming in your project - /drivers/all